themes/my-test-theme-dark/README.md

```diff
@@ -0,0 +1,12 @@
+# KubeColor Themes
+
+If you want to share your custom theme, please open a Pull Request and:
+- explain in the PR what is the goal of the theme (Ex: simple esthetic, disability related, better efficiency to display informations,...)
+- add a foder for your theme into the `themes` folder, and name it with name of your theme and either the `-dark` or `-light` suffix. Ex: `my-test-theme-dark`.
+- a screen capture of the end result as `.jpg` or `.png`
```

Comment:
I think having a sample directory tree as a comment could be useful.

Such as the output of the `tree` command in a code block

Reply:
I think I added that now... please check